### PR TITLE
Blog post on system strings

### LINKED ISSUE #2636: Blog Post: System strings for identifiers and codeable concepts

### PR DESCRIPTION
**Pull request by rahul1**
Fixes #2636

### CODE REVIEW

**Review by codyebberson — APPROVED** [review]
Nice

Super minor, but my first interpretation of the title was `System.String`, which is a weird part of the FHIR spec, because for some reason `System.String` is different than `string`.  Obviously that's not what your post is about 😛

File: packages/docs/blog/2023-08-08-system-strings.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  url: https://github.com/rahul1
+  image_url: https://github.com/rahul1.png
+  tags: [fhir-datastore]
+---
+
+import CodeBlock from '@theme/CodeBlock'
+
+# Demystifying FHIR System Strings
+
+One of the main sources of confusion when starting out with FHIR are `system` strings.
```
Comment (reshmakh) [inline packages/docs/blog/2023-08-08-system-strings.md:16]: One of the main sources of confusion when starting an implementation is with FHIR are `system` strings.

File: packages/docs/blog/2023-08-08-system-strings.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+This field is ubiquitous across FHIR elements, but many developers who are new to healthcare don't understand its purpose or how to set it properly.
```
Comment (reshmakh) [inline packages/docs/blog/2023-08-08-system-strings.md:18]: This field is ubiquitous across FHIR elements, but many developers who are new to healthcare don't understand its purpose or how to set it properly.  (Suggest: adding sentence: Even the [sample data](/docs/tutorials/importing-sample-data) we provide for initial prototyping has many system identifiers, which means they are often needed even for a basic implementation.

File: packages/docs/blog/2023-08-08-system-strings.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+So today, we're going to delve into `system` strings to understand what they're for and how to use them!
+
+System strings are commonly found on two distinct element types:
+
+- [`Identifiers`](/docs/fhir-basics#identifiers-naming-resources)
+- [`CodeableConcepts`](/docs/fhir-basics#codeable-concepts-standarding-data)
+
```
Comment (reshmakh) [inline packages/docs/blog/2023-08-08-system-strings.md:26]: If you have loaded up your sample data, then you can see some examples in the [Patient](app.medplum.com/Patient) resource page for identifiers and [MedicationRequest](app.medplum.com/MedicationRequest) resource page for Codable concepts.

**Review by reshmakh — COMMENTED** [review]
Some feedback mixed across vercel and Github https://medplum-www-git-rahul-blog-system-strings-medplum.vercel.app/blog

**Pull request merged**